wolfcrypt/src/integer.c

```diff
@@ -1996,7 +1996,7 @@ int mp_exptmod_fast (mp_int * G, mp_int * X, mp_int * P, mp_int * Y,
    * one of many reduction algorithms without modding the guts of
    * the code with if statements everywhere.
    */
-  int     (*redux)(mp_int*,mp_int*,mp_digit) = NULL; // cppcheck-suppress nullPointerRedundantCheck // cppcheck 2.6.3 false positive
+  int     (*redux)(mp_int*,mp_int*,mp_digit) = NULL; /* //cppcheck-suppress nullPointerRedundantCheck //cppcheck 2.6.3 false positive */
```

Comment:
As we discovered, the `/* // cppcheck-suppress foo */` code pattern doesn't work.  But we also found that `cppcheck` 2.7 doesn't false-fire on this codepoint, so we just don't need to the suppression at all anymore.

Reply:
(note I left a long followup comment inline in the code, in our conversation around the `XSTRNCASECMP()` macro.  apparently github doesn't copy inline followups into the main conversation page.)

Reply:
Yes, `XSTRNCASECMP()` is a misnomer. Looks more unattractive to change from `strcmp()` to the case-insensitive sibling `strcasecmp()` than the other way around.